test/indexing.jl

```diff
@@ -1299,10 +1299,17 @@ end
         sdf[:, 1] = 10:12
         @test df == DataFrame(a=10:12, b=4:6, c=7:9)
         @test_throws MethodError sdf[:, 1] = ["a", "b", "c"]
-        @test_throws ArgumentError sdf[:, :z] = ["a", "b", "c"]
         @test_throws BoundsError sdf[:, 4] = ["a", "b", "c"]
         @test_throws DimensionMismatch sdf[:, 1] = [1]
         @test_throws MethodError sdf[:, 1] = 1
+        if DataFrames.is_column_insertion_allowed(sdf)
```

Comment:
Any simple way to check this without relying on the internal function? That would allow checking that it's correct too.

Reply:
added